src/Flux/neural_de.jl

```diff
@@ -1,10 +1,10 @@
-function neural_ode(model,x,tspan,
+function neural_ode(model,x,tspan, solver,

```

Comment:
I pulled `solver` out of `args` so it is an explicit field in the struct. I don't know if this is a good idea... @ChrisRackauckas 

Also what are some other examples of `args...` besides  the solver choice?

Reply:
Just use the solver one